pinecone/client.go

```diff
@@ -16,25 +18,41 @@ import (
 
 type Client struct {
 	apiKey     string
```

Comment:
Is apiKey still needed in the `Client` struct?

Reply:
Removed from `Client` as well as `IndexConnection`.